ubuntu_image/assertion_builder.py

```diff
@@ -48,10 +48,12 @@ def populate_rootfs_contents(self):
             dst = os.path.join(self.rootfs, 'system-data')
         for subdir in os.listdir(src):
             # LP: #1632134 - copy everything under the image directory except
-            # /boot which goes to the boot partition.
-            if subdir != 'boot':
-                shutil.move(os.path.join(src, subdir),
-                            os.path.join(dst, subdir))
+            # /boot which goes to the boot partition. Unless this is a uc20
+            # system which has every under "system-seed".
```

Comment:
s/every/everything/ ? i.e. just a typpo, "every" is not a noun.

Reply:
Indeed, silly me. fixed.